Exiled.Events/Patches/Events/Server/KillPlayer.cs

```diff
@@ -0,0 +1,52 @@
+// -----------------------------------------------------------------------
+// <copyright file="KillPlayer.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Events.Server
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    using Exiled.API.Features;
+
+    using HarmonyLib;
+
+    using Mirror;
+
+    using PlayerStatsSystem;
+#pragma warning disable SA1313 // Parameter names should begin with lower-case letter
+
+    /// <summary>
+    /// Prefix of KillPlayer action.
+    /// </summary>
+    [HarmonyPatch(typeof(PlayerStats), nameof(PlayerStats.KillPlayer))]
+    internal class KillPlayer
```

Comment:
You also need to move it to the `Fixes` directory

Reply:
or in `Generic`

Reply:
Done